charts/geoserver/templates/pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.useExisting) -}}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ include "mapproxy.fullname" . }}
```

Comment:
Is `mapproxy` intended here?

Reply:
nope, good catch! fixed in 772e2359fa9d58989ed833893d320b4dda1ad6a0